app/src/main/java/com/x8bit/bitwarden/data/platform/manager/model/NotificationPayload.kt

```diff
@@ -33,39 +37,39 @@ sealed class NotificationPayload {
      */
     @Serializable
     data class SyncFolderNotification(
-        @SerialName("Id") val id: String,
-        @SerialName("UserId") override val userId: String,
+        @SerialName("Id") val folderId: String?,
+        @SerialName("UserId") override val userId: String?,
         @Contextual
-        @SerialName("RevisionDate") val revisionDate: ZonedDateTime,
+        @SerialName("RevisionDate") val revisionDate: ZonedDateTime?,
     ) : NotificationPayload()
 
     /**
      * A notification payload for user-based operations.
      */
     @Serializable
     data class UserNotification(
-        @SerialName("UserId") override val userId: String,
+        @SerialName("UserId") override val userId: String?,
         @Contextual
-        @SerialName("Date") val date: ZonedDateTime,
+        @SerialName("Date") val date: ZonedDateTime?,
     ) : NotificationPayload()
 
     /**
      * A notification payload for sync send operations.
      */
     @Serializable
     data class SyncSendNotification(
-        @SerialName("Id") val id: String,
-        @SerialName("UserId") override val userId: String,
+        @SerialName("Id") val sendId: String?,
+        @SerialName("UserId") override val userId: String?,
         @Contextual
-        @SerialName("RevisionDate") val revisionDate: ZonedDateTime,
+        @SerialName("RevisionDate") val revisionDate: ZonedDateTime?,
     ) : NotificationPayload()
 
     /**
      * A notification payload for passwordless requests.
      */
     @Serializable
     data class PasswordlessRequestNotification(
-        @SerialName("UserId") override val userId: String,
-        @SerialName("Id") val id: String,
+        @SerialName("UserId") override val userId: String?,
+        @SerialName("Id") val loginRequestId: String?,
```

Comment:
@vvolkgang The crash we are seeing in Firebase indicates that the `UserId` and `Id` are both null. Do we know if it is normal for the required data to be omitted from the notification payloads? Is this something that should be addressed on the server side?